samples/dashboard/demos/climate/demo.js

```diff
@@ -1,12 +1,17 @@
 /* eslint-disable prefer-const, jsdoc/require-description */
-// const dataScopes = {
-//     FD: 'Days with fog',
-//     ID: 'Days with ice',
-//     RR1: 'Days with rain',
-//     TN: 'Average temperature',
-//     TX: 'Maximal temperature'
-// };
 const dataPool = new Dashboard.DataOnDemand();
+const dataScopes = {
+    FD: 'Days with Fog',
+    ID: 'Days with Ice',
+    RR: 'Days with Rain',
+    TN: 'Average Temperature',
+    TX: 'Maximal Temperature'
+};
```

Comment:
I see you defined nice names - could we use them in DataGrid too? Instead of abbreviations? 

If it's something more time-consuming than ~15min, I will create a ticket, just let me know, please

Reply:
Please create a ticket. This needs the DataGrid to take column options and we need to create column aliases in the table.